docs/9.0/api/autoColumnSize.md

```diff
@@ -54,7 +54,7 @@ Sampling accepts additional options:
   }
 ```
 
-To configure this plugin see [Options#autoColumnSize](./options/#autocolumnsize).
+To configure this plugin see [autoColumnSize](#options+autocolumnsize).
```

Comment:
The link is broken

Reply:
https://github.com/handsontable/handsontable/pull/8438